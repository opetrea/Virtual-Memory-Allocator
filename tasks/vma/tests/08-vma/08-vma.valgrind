==91853== Memcheck, a memory error detector
==91853== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91853== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91853== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91853== Parent PID: 91852
==91853== 
==91853== 
==91853== HEAP SUMMARY:
==91853==     in use at exit: 0 bytes in 0 blocks
==91853==   total heap usage: 11 allocs, 11 frees, 8,416 bytes allocated
==91853== 
==91853== All heap blocks were freed -- no leaks are possible
==91853== 
==91853== For lists of detected and suppressed errors, rerun with: -s
==91853== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
