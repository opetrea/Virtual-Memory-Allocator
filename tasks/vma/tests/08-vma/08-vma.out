The end address is past the size of the arena
This zone was already allocated.
Total memory: 0xA bytes
Free memory: 0x9 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 0x5 - 0x6
Miniblock 1:		0x5		-		0x6		| RW-
Block 1 end
